sources/packages/backend/apps/api/src/route-controllers/user/user.aest.controller.ts

```diff
@@ -43,6 +47,7 @@ export class UserAESTController extends BaseController {
    * Creates or updates Ministry user information.
    * @param userToken user token information to be updated.
    */
+  @RequiresUserAccount(false)
```

Comment:
For this method, to comply with the below AC I believe we should validate if the necessary information is present (userName, email, lastName) and throw an error to force the redirect to the login in the below area in the Web.
![image](https://github.com/user-attachments/assets/59b8ad31-36fb-4b30-8f2d-0754fdd142db)
```
If a user logs in and has no user profile or does not have enough information to create a user profile (example: missing email) they should be redirected to the login page and receive the same message as an unauthorized user with no user profile.
```
Please let me know if I am missing something.